### PR TITLE
Jp update

### PR DESCRIPTION
**Pull request by PilgrimMemoirs**
Scale down requirements of project into a single page with focus on specific layout techniques,  styling with CSS and making CSS organized and readable. 

Still needs wireframe and updated index.html

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,40 +1,37 @@
...
-
-
-<img src="assets/imgs/profile_layout.jpg" alt="profile layout" width="250px"/>
-
-## Wave 3
-### Refactor
-
-Look over your CSS and identify any sets of declarations that are being used a lot. Refactor you CSS to be more dry by utilizing multiple selectors or creating a class.
+### Optional Enhancements
+- Add parallax scrolling to the hero banner
```
Comment (droberts-sea) [inline README.md]: For some reason github renders this last line as header-sized.

**Review by droberts-sea — CHANGES_REQUESTED** [review]
Looks good!

**Review by droberts-sea — APPROVED** [review]

**Comment by CheezItMan** [comment]
🚢

File: README.md [hunk 2]
```diff
@@ -1,40 +1,45 @@
...
+- Add Custom Styling
 
-Use the drawings below to make the provided html code match the layouts. These drawings are called wireframes, they are like a blueprint for your site's layout design. It highlights the layout without focusing on any of the design specifics (like color, fonts etc.).
+## Baseline
+Using CSS, construct the layout of the site according to the drawing below.
+![Meowspace Wireframe](./assets/meowspace.png)
 
 
-First, use CSS to rearrange the layout to match the drawings as much as possible (it doesn't need to be perfect).  Only then can you jazz up the site's design with more pleasing colors, fonts and small touches.
+This drawings is called a wireframe. Wireframes are like a blueprint for your site's layout design. It highlights the layout without focusing on any of the design specifics (like color, fonts etc.). The areas that have an giant 'x' crossed through it are meant to represent the areas that have an image. **Hint:** There are two sections in this wireframe that have images as backgrounds.
```
Comment (kariabancroft) [inline README.md]: Typo: drawing vs drawings

File: README.md [hunk 3]
```diff
@@ -1,40 +1,45 @@
...
-  - WITHOUT moving any content around
-  - You can add sectioning elements, ID's and classes and necessary
+**You can make only make limited changes to the HTML**
+  - You cannot move any content around in the HTML
+  - You can add sectioning elements to wrap around content
+  - You can add ID's and classes to elements, as necessary
 
+## Primary Requirements
 
+- Add normalize.css
```
Comment (kariabancroft) [inline README.md]: Does this mean to add a new CSS file with this name? There is a `main.css` file in the project structure so i'm not sure if they're supposed to use that?

**Review by kariabancroft — COMMENTED** [review]
Looks good! Just one specific question about the instructions

File: README.md (hunk 3 above)
Comment (PilgrimMemoirs) [reply README.md]: I was referring to this https://necolas.github.io/normalize.css/

I know i brought it up with C6, but I can through in a quick spiel about why it is and why to use it.

**Pull request merged**